app/views/layouts/application.html.erb

```diff
@@ -56,13 +56,16 @@
       orcid_url: '<%= raw(ORCID_URL) %>',
       ror_url: '<%= raw(ROR_URL) %>',
       datacite: <%= raw(
-        ['ContributorType', 'RelationType', 'RelatedIdentifierType'].map { |name|
+        ['RelationType', 'RelatedIdentifierType'].map { |name|
           [
             name,
             Datacite::Mapping.const_get(name).to_a.map(&:value)
           ]
         }.to_h.to_json
       ) %>,
+      dataciteContributorType: <%= raw(
+        Datacite::Mapping::ContributorType.to_a.map { |x| {key: x.key, value: x.value.titleize} }.to_json
+      ) %>
```

Comment:
This is the big change in this PR. We now gather both the key and the human value (e.g. "CONTACT_PERSON" and "Contact Person") rather than always using "ContactPerson"